plugins/export.js

```diff
@@ -11,4 +11,5 @@ window.Redux = require('redux');
 window.ReactRedux = require('react-redux');
 window.ReactBootstrap = require('react-bootstrap');
 window.PostUtils = {formatText, messageHtmlToComponent};
-window.PropTypes = require('prop-types');
\ No newline at end of file
+window.PropTypes = require('prop-types');
+window.PDFJS = require('pdfjs-dist');
```

Comment:
I think this is only required for certain libraries because they need to be consistent with the webapp or there can only be one loaded on the window at a time. I don't think pdfjs falls into this category. 

What's your reasoning for exporting this?

Reply:
1. If `pdfjs-dist` is installed via npm and imported in a plugin component, the following warning is shown while building the plugin:
```
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  main.js (437 KiB)
  1.main.js (698 KiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (437 KiB)
      main.js
```
By exposing `PDFJS` using Webpack externals, we can avoid the above warning.

2. I was running into several issues with using PDFJS in a plugin without the above change.
By exposing the `PDFJS` object to the window, it would be easier for plugin developers to render PDFs.

Reply:
1. Yes, I guess it's a bit large, maybe worth it to export.
2. What where those issues?

Reply:
2.> `Uncaught TypeError: r(...) is not a function` would be logged to the console.

![Screenshot from 2019-03-26 12-40-13](https://user-images.githubusercontent.com/35728906/54977881-b035a200-4fc4-11e9-8acf-06733b381e01.png)

You can go to the current master branch and see this if you run `npm i -S pdfjs-dist` and add `import PDFJS from 'pdfjs-dist';` in any plugin.
I have tried with different versions of PDFJS. Couldn't find much help from the internet about this issue as well.

But this does get resolved by eporting PDFJS to the window from mattermost-webapp to be used in plugins.

Reply:
Alright I think this is fine then.